src/stepDefinitions/filesContext.js

```diff
@@ -103,3 +129,23 @@ Given('user {string} has created the following files', async function (userId, e
   }
   return client
 })
+
+When('user {string} has renamed the following file', function (user, table) {
+  const fromName = table
+    .hashes()
+    .map((data) => data['from-name-parts'])
+    .join('')
+  const toName = table
+    .hashes()
+    .map((data) => data['to-name-parts'])
+    .join('')
+  return webdav.move(user, fromName, toName)
+})
+
+Given(
+  'user {string} has locked file/folder {string} setting following properties',
```

Comment:
```suggestion
  'user {string} has locked file/folder {string} setting the following properties',
```

In oC10 core this step has "the" in the text. Can we add that here now? (And adjust the words in web...)